alloc/src/chain.rs

```diff
@@ -239,6 +239,7 @@ unsafe impl<A: Allocator + Clone> Allocator for ChainAllocator<A> {
         debug_assert!(chain_node.remaining_capacity() >= layout.size());
 
         let result = chain_node.linear.allocate(layout);
+        #[cfg(not(fuzzing))]
```

Comment:
Does `cargo-bolero` or sets `fuzzing` when it's invoking the tests?

Reply:
appears to